### PR TITLE
Replace Calypso Mocks

### LINKED ISSUE #79: Revamp Calypso/Mock
### Description

Right now, if we get a CAN ID we dont recognize, we send a blank message called Calypso/Mock.  Also, if we get a message shorter than expected, we silently fail.

### Acceptance Criteria

Carry out proposed solution or explain alternative solution.

### Proposed Solution

Cases of an ID which does not match anything should be revamped to include the CANID of the unrecognized message as the payload, the topic of which will be Calypso/MiscMessage or something IDC.  

Cases of messages with a too small length should also be debugged, either by printing an error or sending a CAN message.

### PR DESCRIPTION
**Pull request by jr1221**
## Changes

_Explanation of changes goes here_

Remove encode and decode mock from the daedelus layer.

Replace them with:
- Decode Mock: `Calypso/Unknown`, value= the ID of the message that could not be decoded
- Encode Mock: `Calypso/Bidir/Unknown` value=the number of times that an invalid encode key is recieved.

## Notes

_Any other notes go here_

## Test Cases

- Case A
- Edge case
- ...

## To Do

_Any remaining things that need to get done_

- [ ] item 1
- [ ] ...

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please reach out to your Project Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] No merge conflicts
- [ ] All checks passing
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes #79

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -197,16 +196,30 @@ fn read_siren(pub_path: &str, send_map: Arc<RwLock<HashMap<u32, EncodeData>>>) -
...
+                        send_map
+                            .write()
+                            .expect("Could not modify send messages!")
+                            .insert(ret.id, ret);
+                    }
+                    None => {
+                        let id: u32 = 0x7FF;
+                        let mut send_map_writable =
+                            send_map.write().expect("Could not modify send messages!");
+                        let cnt = match send_map_writable.get(&id) {
```
Comment (harrison-e) [inline src/main.rs:211]: taboo variable name!

**Review by harrison-e — APPROVED** [review]
lgtm
